setup/www/resources/config/nightly-builder-v8-canary.json.j2

```diff
@@ -0,0 +1,10 @@
+{
+  "jenkinsToken": "{{ jenkins_token }}",
+  "jenkinsJobUrl": "https://ci-release.nodejs.org/job/iojs+release",
+  "jenkinsCrumbUrl": "https://ci-release.nodejs.org/crumbIssuer/api/json",
+  "githubAuthUser": "{{ github_auth_user }}",
+  "githubAuthToken": "{{ github_auth_token }}",
+  "githubOrg": "nodejs",
+  "githubRepo": "node-v8",
+  "githubScheme": "https://github.com/",
+}
```

Comment:
Sublime doesn't add newlines at the end of files? Huh...

AIUI the problem is that some tools (editors or git or something) require a newline at the end. It's never caused an issue for me so 🤷‍♂️ .